src/slist.c

```diff
@@ -1 +1,324 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdbool.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/xlogging.h"
+
 #include "c_util/slist.h"
+
+bool slist_initialize(PSINGLYLINKEDLIST_ENTRY list_head)
```

Comment:
>bool [](http://example.com/codeflow?start=0&length=4)

This is better IMO as an int.
We got used to have int returns that return 0/non-zero, because we can actually code in the non-zero failure code the line number where the failure happens.

Which sometimes matters quite a lot.

Reply:
done